src/Serilog.AspNetCore/RequestLoggingOptions.cs

```diff
@@ -12,17 +12,37 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using Serilog.Events;
 using System;
+using System.Net;
 
 namespace Serilog
```

Comment:
We can move this to `Serilog.AspNetCore` to avoid polluting a commonly-imported namespace; the type name doesn't have to be in scope in order for consuming code to call `UseSerilogRequestLogging(opts =>..`.

Reply:
Moved the options under `Serilog.AspNetCore`. Not sure to understand what you mean by 

> the type name doesn't have to be in scope in order for consuming code to call UseSerilogRequestLogging(opts =>...

Reply:
It means that it should have the same namespace to avoid writing additional `using` statement.

Reply:
Thanks @sungam3r; I meant the opposite, in this case - there's no need for a `using` statement even if the type is in a different namespace, since the members are accessed via the `options` action argument (sorry if I was unclear, it looks like the correct update has been made).

Reply:
ok